tests/test_configurations.py

```diff
@@ -47,6 +47,42 @@ def tearDown(self):
                 print(e)
 
 
+class SMRFTestCaseLakes(unittest.TestCase):
+    """
+    The base test case for SMRF that will load in the configuration file and store as
+    the base config. Also will remove the output directory upon tear down.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Runs the short simulation over reynolds mountain east
+        """
+
+        cls.test_dir = os.path.join('tests', 'Lakes')
+
+        # check whether or not this is being ran as a single test or part of the suite
+        cls.config_file = os.path.join(cls.test_dir, 'config.ini')
+
+        # read in the base configuration
+        cls.base_config = get_user_config(cls.config_file, modules = 'smrf')
```

Comment:
Little late to the party and I would have moved this into its own file and called the file the same as the class. Makes it easier to find the declaration in the future.
Also the description seems outdated and the commented out code below should ideally not be there.

Reply:
This is a true patch for this release. We'll need to make these changes into the master branch and will make it look a lot prettier!

Reply:
Got it. Didn't look close enough that this is a release branch and not master 👍